src/Service/LabelNameExtractor.php

```diff
@@ -13,16 +13,11 @@
  */
 class LabelNameExtractor
 {
-    private static array $labelAliases = [
+    private const array LABEL_ALIASES = [
         'bridge\doctrine' => 'DoctrineBridge',
-        'bridge/doctrine' => 'DoctrineBridge',
```

Comment:
Why removing them ?

Reply:
DX ? Having 5 felt easier to read/maintain, but this was not really in the scope of the PR i must admit. Should i revert ?

Reply:
Let's revert

Reply:
Just read the rest of the code, I think it's fine.